ui/shared/DropdownMenuItems.jsx

```diff
@@ -4,27 +4,46 @@ import { DropdownMenu, DropdownItem } from 'reactstrap';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCheck } from '@fortawesome/free-solid-svg-icons';
 
+const createDropdownItem = (item, namespace, updateData, selectedItem) => {
+  return (
+    <DropdownItem
+      tag="a"
+      key={`${namespace}${item}`}
+      onClick={() => updateData(item)}
+    >
+      <FontAwesomeIcon
+        icon={faCheck}
+        className={`mr-1 ${selectedItem === item ? '' : 'hide'}`}
+        title={selectedItem === item ? 'Checked' : ''}
+      />
+      {item}
+    </DropdownItem>
+  );
+};
+
 const DropdownMenuItems = ({
   selectedItem,
   updateData,
   options,
+  pinned = [],
```

Comment:
This should be a default prop rather than assigning a default value of `[]`.

Reply:
I don't believe I've used default props before, you mean something like this? https://reactjs.org/docs/react-component.html#defaultprops